src/app/components/AddButton/AddButton.tsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import classes from './AddButton.module.css';
 import { GiLightBackpack } from 'react-icons/gi';
-import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 
 export default function AddButton(): JSX.Element {
+  const navigate = useNavigate();
   return (
     <div
       data-augmented-ui="all-hexangle-up border"
       className={classes.button_area_div}
     >
-      <Link to="/input">
-        <button className={classes.add_button_symbol}>
-          <GiLightBackpack size={42} />
-        </button>
-      </Link>
+      <button
+        onClick={() => navigate('/Input')}
```

Comment:
The button itself should be 'stupid'. Meaning the component should only know that it will get a function, but not which one. This would be handled on the page itself.
For now I think this is fine, just something to consider 🤓